app/src/main/java/nerd/tuxmobil/fahrplan/congress/details/SessionDetailsFragment.kt

```diff
@@ -265,9 +271,12 @@ class SessionDetailsFragment : Fragment() {
         }
     }
 
-    private fun TextView.applyText(typeface: Typeface, text: String) {
+    private fun TextView.applyText(typeface: Typeface, text: String, contentDescription: String? = null) {
         this.typeface = typeface
         this.text = text
+        if (contentDescription != null) {
```

Comment:
What is the advantage of your suggestion?

Reply:
It's just a more programmatic in style and adds not so much here aside to verifying that `contentDescription` is indeed a String (or it could be CharSequence and nothing else). Hence even if you would change the method signature into `contentDescription: Any` this still would be valid and working while `!= null` will not.

Reply:
The `is` operator does make sense to me if the condition is more complex because the smart cast would allow the right hand side of the condition to benefit from knowing the type. In the given case I the condition is very simple.

I checked what the Kotlin is decompiled to. No surprise there. However, Lint immediately outputs that a null check is good enough instead of the `inInstance` check - see the screenshot.

![instanceof](https://user-images.githubusercontent.com/144518/144760365-13124a7e-b26c-4e61-9d5e-a31629a6d911.png)

Regarding the property - actually `CharSequence` is used for the parameter, see `android.view.View#setContentDescription`.

:x: In sum I wouldn't change the condition here. 

Whenever it makes sense I keep it in mind as an option. I will look through the cases in your #422 review, too.